### ISSUE TITLE
Windows7 KB3045999 is not availiable on microsoft offical website

### ISSUE THREAD

**Issue by burningheart** [keybase/keybase-issues#2629]
here is a problem that windows need a support of KB3045999 but i cant install it .
so i cant install windows keybase client

**Comment by zQueal** [comment]
I'm not sure if this is accurate. I do not have this update installed and Keybase is working fine for me. What makes you think you require this update to use Keybase?

**Comment by taruti** [comment]
cc @zanderz

**Comment by zanderz** [comment]
Yes, without an install log (see `%temp%\Keybase*log` and `%temp%\Dokan*log`), I can't tell what the problem could be, since I see no dependency on KB3045999 in our sources. What is the exact symptom you see? Can you provide a screenshot and/or the logs I mentioned?
